lib/pdk/util.rb

```diff
@@ -196,7 +197,7 @@ def in_module_root?(path = Dir.pwd)
     # @return [Hash, nil] subset of text as Hash of first valid JSON found, or nil if no valid
     #   JSON found in the text
     def find_first_json_in(text)
-      find_valid_json_in(text)
+      PDK::Util::JSONFinder.new(text).objects.first
```

Comment:
nit: Should this just be `find_all_json_in(text).first` ?

Reply:
:+1: